en/components/select.md

```diff
@@ -34,20 +35,74 @@ import { IgxSelectModule } from 'igniteui-angular';
 export class AppModule {}
 ```
 
-In your class you need to have a collection of the items that you want to display when the drop-down opens:
-```ts
+Then all that is needed is to define an `igxSelect` and several items.
+```html
+<!-- define items declaratively -->
+<igx-select>
+    <igx-select-item value="Orange">Orange</igx-select-item>
+    <igx-select-item value="Apple">Apple</igx-select-item>
+    <igx-select-item value="Banana">Banana</igx-select-item>
+</igx-select>
+```
+
+Alternatively, another way to do it would be to simply pass in a collection of the items that we want to display using [*ngForOf*](https://angular.io/api/common/NgForOf) directive.
+First, specify a collection of the items that you want to display when the drop-down opens:
+```typescript
 public items: string[] = ["Orange", "Apple", "Banana", "Mango"];
 ```
 
 Then in your template you need to bind it with said items like so:
 ```html
-<igx-select>
+<igx-select [(ngModel)]="selected">
+    <igx-select-item *ngFor="let item of items" [value]="item">
+        {{item}}
+    </igx-select-item>
+</igx-select>
+```
+Additionally, we are binding via `ngModel` so there is initial `igxSelect` value set.
```

Comment:
I think this should be a separate section with a bit more explanation, the `ngModel` binding is not to be taken lightly. 
Alternatively, this could be under the "Features" section.
Thoughts?

Reply:
I can see the benefit from a separate section regarding two-way-binding under Features. However, as the document is already containing too much sections & samples we may consider leaving the current state of it. 

I have made a bit of update on the related two-way binding wording though.